libs/sdk-backend-bear/src/backend/workspace/metadata/index.ts

```diff
@@ -220,4 +224,70 @@ export class BearWorkspaceMetadata implements IWorkspaceMetadata {
             }),
         );
     }
+
+    public getUriIdentifierObjects = async (identifiers: Identifier[]): Promise<IUriIdentifierObject[]> => {
```

Comment:
IMO, use the name `get...By...` is easier to understand

Reply:
changed